module/data/item/talent.mjs

```diff
@@ -11,7 +11,14 @@ export default class TalentData extends AbilityTemplate.mixin(
     /** @inheritDoc */
     static defineSchema() {
         return this.mergeSchema( super.defineSchema(), {
-            
+            talentType: new foundry.data.fields.StringField( {
```

Comment:
Can we find a more descriptive name for that? Maybe something like "source", "sourceType", "learnedThrough" or something like that?

Reply:
okidoki